.jshintrc

```diff
@@ -38,7 +38,7 @@
     "debug"         : false,     // true: Allow debugger statements e.g. browser breakpoints.
     "eqnull"        : true,      // true: Tolerate use of `== null`
     "es5"           : false,     // true: Allow ES5 syntax (ex: getters and setters)
-    "esnext"        : false,     // true: Allow ES.next (ES6) syntax (ex: `const`)
+    "esnext"        : true,      // true: Allow ES.next (ES6) syntax (ex: `const`)
```

Comment:
You can replace `es5` and `esnext` with `"esversion"     : 6,` which seems nicer (esnext will sound stupid once es7 is out... `esnextnext`?)